services/app-api/utils/formTemplates/formTemplates.ts

```diff
@@ -325,3 +330,28 @@ const makePCCMTemplateModifications = (reportTemplate: ReportJson) => {
   }
   programTypeQuestion.props!.disabled = true;
 };
+
+export const makeTopicXIIModifications = (reportTemplate: ReportJson) => {
+  /*
+   *   Find Question C1.XII.4, nest 11.a within 10.b
+   *   also remove the response "No deficiencies identified" under question 11
+   */
+  const question11 =
```

Comment:
`reportTemplate.routes[2].children![6].form!.fields[0].props?.choices[1].children[6]` looks very scary to me. Would prefer if we could know what these indices mean. Such as:

```
const topicRoute = reportTemplate.routes.find(val === matchingRoute) // instead of [2]
const topicChild = topicRoute.find(val === matchingChild) // instead of [6]

etc.
```

At the least, it seems that `reportTemplate.routes[2].children![6].form!.fields[0].props?.choices[1]` could be assigned to a variable because it's reused.